src/main/webapp/app/entities/participation/participation.component.html

```diff
@@ -43,7 +43,12 @@ <h2>
             <tbody>
                 <tr *ngFor="let participation of participations | sortBy: predicate:reverse; trackBy: trackId">
                     <td>
-                        <a routerLink="/participation/{{ participation.id }}">{{ participation.id }}</a>
+                        <div *ngIf="exercise.isAtLeastInstructor; else displayId">
```

Comment:
**Optional Suggestion:** Here you could use a `<ng-container>` to keep the produced HTML slender.